yjit_iface.c

```diff
@@ -481,6 +481,7 @@ rb_yjit_compile_iseq(const rb_iseq_t *iseq, rb_execution_context_t *ec)
 {
 #if OPT_DIRECT_THREADED_CODE || OPT_CALL_THREADED_CODE
     RB_VM_LOCK_ENTER();
+    // TODO: I think we need to stop all other ractors here
```

Comment:
You might be right. I guess if other ractors are running, they might start compiling the same code at the same time-ish?

Reply:
Yeah. Actually, I'm not sure if we need to stop all other ractors, but we should probably make sure only one ractor wins the race.

Reply:
Doesn't seem to be causing issue at the moment though. Maybe because we are not patching native code when generating entry points.